recipes/twitchtv-libsoundtrackutil/all/CMakeLists.txt

```
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 3.1)
+project(cmake_wrapper)
+include(conanbuildinfo.cmake)
+
+conan_basic_setup(TARGETS)
+add_subdirectory("source_subfolder")
```

Whitespace to make it easier to search and replace later (optional)

```suggestion
cmake_minimum_required(VERSION 2.8.12)
project(cmake_wrapper)

include("conanbuildinfo.cmake")
conan_basic_setup()

add_subdirectory(source_subfolder)
```